src/main/java/dev/jbang/dependencies/DependencyUtil.java

```diff
@@ -111,9 +81,10 @@ public static ModularClassPath resolveDependencies(List<String> deps, List<Maven
 				repos.stream().map(m -> m.toString()).collect(Collectors.joining(", "))));
 
 		String depsHash = String.join(CP_SEPARATOR, depIds);
-		if (!transitivity) { // the cached key need to be different for non-transivity
-			depsHash = "notransitivity-" + depsHash;
-		}
+		// if (!transitivity) { // the cached key need to be different for
```

Comment:
why don't we need this anymore ?

Reply:
The methods was only ever called with `true` so it obviously wasn't used anymore. I commented it out exactly for this reason: that we might take a closer look. Btw that code has been like that for quite some time now, it's not a recent change.